lib/spoom/sorbet/translate/sorbet_sigs_to_rbs_comments.rb

```diff
@@ -97,19 +102,39 @@ def visit_constant_write_node(node)
 
         #: (Prism::ClassNode | Prism::ModuleNode | Prism::SingletonClassNode) { -> void } -> void
         def visit_scope(node, &block)
-          @nesting << node
+          old_class_annotations = @class_annotations
+          @class_annotations = []
           old_type_members = @type_members
           @type_members = []
+          old_extend_t_helpers = @extend_t_helpers
+          @extend_t_helpers = []
+          old_extend_t_generics = @extend_t_generics
+          @extend_t_generics = []
+          old_seen_mixes_in_class_methods = @seen_mixes_in_class_methods
+          @seen_mixes_in_class_methods = false
```

Comment:
nit: Personally, I find this style easier to follow:

```suggestion
          old_class_annotations, @class_annotations = @class_annotations, []
          old_type_members, @type_members = @type_members, []
          old_extend_t_helpers, @extend_t_helpers = @extend_t_helpers, []
          old_extend_t_generics, @extend_t_generics = @extend_t_generics, []
          old_seen_mixes_in_class_methods, @seen_mixes_in_class_methods = @seen_mixes_in_class_methods, false
```

Reply:
Ah no worries then it's good to ship already 👍